src/components/subheader/constants/subheader.general.js

```diff
@@ -0,0 +1,7 @@
+export const INITIAL_STATE = {
+    subheaderData : null,
+    isLoading : true,
+    error : null
+}
+
+export const subheaderDataUrl = 'https://json-server-swiggy.herokuapp.com/subheaderData';
```

Comment:
don't keep this here, because the baseUrl(origin) of this will be same throughout the application so keep it at the root level of the project and you can have the path name  or may be ID here

Reply:
and also have default values like RESTAURANT_ID